### ISSUE TITLE
Awesome bar: change the tag match syntax

### ISSUE THREAD

**Issue by benini** [benini/scid#117]
Intead of:
_white carlsen_
or
_white "Carlsen, Magnus"
it may be better to change the syntax to:
white=carlsen
white="Carlsen, Magnus"

Essentially the part before '=' is matched against the exact tag name.

**Comment by nicktmro** [comment]
Silly question, but what is the `Awesome bar`? Is it the search box at the bottom of the Game List? (Couldn't find a reference in the wiki or the guide)

**Comment by benini** [comment]
Yes. I wrote it many years ago when Firefox created the "awesome bar". The idea was to create something similar, but I don't think it is really possible with Tk. It is just a filter bar ( https://github.com/benini/scid/wiki/Filter-games#filter-bar ).
